js/src/views/Account/Transactions/Transaction/transaction.js

```diff
@@ -160,7 +161,14 @@ export default class Transaction extends Component {
     const { api } = this.context;
     const ether = api.util.fromWei(value);
 
-    return `${ether.toFormat(5)}`;
+    if (ether.eq(0)) {
+      return { value: null, token: null };
+    }
+
+    return {
+      value: `${ether.toFormat(5)}`,
+      token: <small>ΞTH</small>
```

Comment:
being really picky – As of HTML5, `<small>` has a semantical meaning and should not be used for stylistic purposes. Use `<span`.

(also mentioned in https://github.com/ethcore/parity/pull/2099#discussion_r79003278)